packages/react-native/Libraries/NativeAnimation/Nodes/RCTColorAnimatedNode.mm

```diff
@@ -21,7 +21,9 @@ - (void)performUpdate
   RCTValueAnimatedNode *bNode = (RCTValueAnimatedNode *)[self.parentNodes objectForKey:self.config[@"b"]];
   RCTValueAnimatedNode *aNode = (RCTValueAnimatedNode *)[self.parentNodes objectForKey:self.config[@"a"]];
 
-  _color = RCTColorFromComponents(rNode.value, gNode.value, bNode.value, aNode.value);
+  _color = self.config[@"space"] != nil
+    ? RCTColorFromComponents(rNode.value * 255, gNode.value * 255, bNode.value * 255, aNode.value) 
```

Comment:
we should pass the space, no?

Reply:
@cipolleschi Yeah I believe that's the better solution. Is there any value in converting back and forth from components to integer at all anymore? Or should we just create the UIColor from the color components directly?

Reply:
No, I think that at this point we can keep the UIColors. I have no idea why we used to use ints for colors, but it looks a very bad API to me.